samples/deployment/test-network/Makefile

```diff
@@ -6,9 +6,12 @@
 TOP = ../../..
 include $(TOP)/build.mk
 
-
-TEST_CC_ID ?= echo
-TEST_CC_PATH ?= ${FPC_PATH}/samples/chaincode/${TEST_CC_ID}
+ifeq ($(CC_ID),)
+ $(error CC_ID is not set) && exit 1;
+endif
+ifeq ($(CC_PATH),)
+ $(error CC_PATH is not set) && exit 1;
+endif
```

Comment:
```suggestion
ifeq ($(CC_ID),)
	$(error CC_ID is not set)
endif
ifeq ($(CC_PATH),)
 	$(error CC_PATH is not set)
endif
```

Reply:
it seems that the `exit 1` is not needed as the error command already exits make according to https://www.gnu.org/software/make/manual/html_node/Make-Control-Functions.html

Reply:
You are right. Made the change and tested.